### ISSUE TITLE
Focal minimal pre-release 0.10.12: gitlab-ci-linux-build-184 does not boot Rock64 v2

### ISSUE THREAD

**Issue by gusarg81** [ayufan-rock64/linux-build#445]
Hi,

Today I've tested this pre-release but Rock64 (v2.0) does not boot at all.

**Comment by clach04** [comment]
This may be related to issue #439 (for RockPro64) - there is a workaround.

For rock64 known issues #426 and #428- with sort of a workaround (more of a hack), and potentially a fix for Armbian bug https://armbian.atlassian.net/browse/AR-282 as part of https://www.armbian.com/newsflash/armbian-20-08-caple/

**Comment by gusarg81** [comment]
Since yesterday tested with armbian 5.7 for rock64 and boot normally with emmc.

**Comment by clach04** [comment]
@gusarg81  being slightly cheeky, if you have a few mins please could you check out https://github.com/MichaIng/DietPi/issues/2588 and run the script there and respond? I'm not in front of my device so I can't test this out for a while :(

* https://github.com/MichaIng/DietPi/issues/2588#issuecomment-642601939 is the details they asked for
* https://github.com/MichaIng/DietPi/issues/2588#issuecomment-642747419 is the script I created (and ran with non-armbian installation)  -so easiest to copy/paste and run this

**Comment by gusarg81** [comment]
You mean this one? https://forum.armbian.com/topic/9310-rk3328-media-script-rock64-renegade/

I have the server version installed, not the desktop one (Armbian focal 5.7 kernel).

**Comment by clach04** [comment]
> 
> 
> You mean this one? https://forum.armbian.com/topic/9310-rk3328-media-script-rock64-renegade/
> 
> I have the server version installed, not the desktop one (Armbian focal 5.7 kernel).

No, the diagnostic only script at the head of https://github.com/MichaIng/DietPi/issues/2588#issuecomment-642747419:

diag_script_issues_2588.bash

    #/bin/bash

    # for https://github.com/MichaIng/DietPi/issues/2588#issuecomment-642601939


    echo    Current GPU DevFreq governors:

    ls -l /sys/class/devfreq/
    cat /sys/class/devfreq/ff9a0000.gpu/governor # should return error as ROCKPro64 only
    cat /sys/class/devfreq/ff300000.gpu/governor
    cat /sys/class/devfreq/dmc/governor


        echo Current SMP affinities:

    cat /proc/interrupts
    while read -r irq cpu0 cpu1 cpu2 cpu3
    do
    [[ $icq =~ ahci|echi|ohci|xchi|eth|'0000:01:' ]] && echo -n "$icq = " && cat /proc/irq/${irq%:}/smp_affinity
    done < /proc/interrupts

    echo Onboard Ethernet RPS:

    cat /sys/class/net/eth0/queues/rx-0/rps_cpus
    cat /proc/sys/net/core/rps_sock_flow_entries
    cat /sys/class/net/eth0/queues/rx-0/rps_flow_cnt

    echo I take out CPU governor since we already cover this with dietpi-config/dietpi-set_cpu.

**Comment by gusarg81** [comment]
Ok! Missed that one:

> Current GPU DevFreq governors:
> total 0
> lrwxrwxrwx 1 root root 0 ago 24 10:11 dmc -> ../../devices/platform/dmc/devfreq/dmc
> lrwxrwxrwx 1 root root 0 ago 24 10:11 ff300000.gpu -> ../../devices/platform/ff300000.gpu/devfreq/ff300000.gpu
> lrwxrwxrwx 1 root root 0 ago 24 10:11 ff360000.rkvdec -> ../../devices/platform/ff360000.rkvdec/devfreq/ff360000.rkvdec
> cat: /sys/class/devfreq/ff9a0000.gpu/governor: No such file or directory
> simple_ondemand
> dmc_ondemand
> Current SMP affinities:
>            CPU0       CPU1       CPU2       CPU3       
>   1:          0          0          0          0     GICv2  29 Edge      arch_timer
>   2:    5137860    5138634    4931359    5213396     GICv2  30 Edge      arch_timer
>   3:          0          0          0          0     GICv2  27 Edge      kvm guest timer
>  12:          0          0          0          0     GICv2  90 Level     rockchip_thermal
>  13:          9          0          0          0     GICv2  89 Level     serial
>  14:          0          0          0          0     GICv2  68 Level     ff150000.i2c
>  15:       1962          0          0          0     GICv2  69 Level     ff160000.i2c
>  17:          0       2271          0          0     GICv2  82 Level     rk_pwm_irq
>  18:        232          0          0          0     GICv2  32 Level     ff1f0000.dmac
>  19:          0          0          0          0     GICv2  33 Level     ff1f0000.dmac
>  20:          0          0          0          0     GICv2 122 Level     Mali_GP
>  21:          0          0          0          0     GICv2 119 Level     Mali_GP_MMU
>  22:          0          0          0          0     GICv2 125 Level     Mali_PP_Broadcast
>  23:          0          0          0          0     GICv2 120 Level     Mali_PP0
>  24:          0          0          0          0     GICv2 121 Level     Mali_PP0_MMU
>  25:          0          0          0          0     GICv2 123 Level     Mali_PP1
>  26:          0          0          0          0     GICv2 124 Level     Mali_PP1_MMU
>  27:          0          0          0          0     GICv2  41 Level     ff350000.vpu_service, ff351000.avsd
>  29:          0          0          0          0     GICv2  39 Level     ff360000.rkvdec
>  31:          0          0          0          0     GICv2 127 Level     ff330000.h265e
>  33:          0          0          0          0     GICv2 129 Level     ff340000.vepu
>  35:        252          0          0          0     GICv2  64 Level     ff370000.vop
>  36:          0          0          0          0     GICv2  65 Level     rga
>  37:          0          0          0          0     GICv2  63 Level     ff3a0000.iep
>  38:       1571          0          0          0     GICv2  67 Level     ff3c0000.hdmi, dw-hdmi-cec
>  40:          0          0          0          0     GICv2 115 Level     ff430000.hdmiphy
>  41:          0          0          0          0     GICv2 109 Level     rockchip_u3phy
>  42:    3269348          0          0          0     GICv2  46 Level     dw-mci
>  43:          0          0          0          0     GICv2  44 Level     dw-mci
>  44:    2573247          0          0          0     GICv2  56 Level     eth0
>  45:          0          0          0          0     GICv2  55 Level     ff580000.usb, dwc2_hsotg:usb1
>  46:          0          0          0          0     GICv2  48 Level     ehci_hcd:usb2
>  47:          0          0          0          0     GICv2  49 Level     ohci_hcd:usb3
>  48:          0          0          0          0     GICv2  25 Edge      vgic
> 123:          0          0          0          0     gpio2   6 Level     rk805
> 181:          0          0          0          0     GICv2  94 Level     rockchip_usb2phy
> 182:          0          0          0          0     GICv2  93 Level     rockchip_usb2phy
> 183:          0          0          0          0     GICv2  99 Level     xhci-hcd:usb4
> 184:          0          0          0          0     rk805   0 Edge      rk8xx_pwrkey_fall
> 189:          0          0          0          0     rk805   5 Edge      RTC alarm
> 191:          0          0          0          0     rk805   7 Edge      rk8xx_pwrkey_rise
> IPI0:    473088     863093     469116     396165       Rescheduling interrupts
> IPI1:        48     254200     957411     178715       Function call interrupts
> IPI2:         0          0          0          0       CPU stop interrupts
> IPI3:         0          0          0          0       CPU stop (for crash dump) interrupts
> IPI4:         0          0          0          0       Timer broadcast interrupts
> IPI5:         0          0          0          0       IRQ work interrupts
> IPI6:         0          0          0          0       CPU wake-up interrupts
> Err:          0
> Onboard Ethernet RPS:
> 7
> 32768
> 32768
> I take out CPU governor since we already cover this with dietpi-config/dietpi-set_cpu.

Just a note, that I switched to legacy kernel on this Armbian Focal because I need Hardware encoding to work with FFmpeg.
